src/platform/testing/unit/schemaform-utils.jsx

```diff
@@ -214,7 +215,13 @@ export function getFormDOM(form) {
   };
 
   formDOM.submitForm = () => {
-    submitForm(form);
+    if (form?.container) {
```

Comment:
I tweaked this helper to support return values from both renderIntoDocument and RTL's render, so that I didn't have to update every test using this. The associated functions on `this` are generally just DOM checks and work fine with regular DOM nodes.